src/lib/components/ContentContainer.svelte

```diff
@@ -1,8 +1,12 @@
 <div
-    class="flex flex-col items-center justify-between gap-12 rounded-3xl bg-warm-white px-8 py-8 md:px-32 md:py-16 lg:flex-row"
+    class="flex flex-col items-center justify-between gap-12 rounded-3xl bg-[#f6f6f6] px-8 py-8 md:px-32 md:py-16 xl:flex-row dark:bg-[#1d1d1d]"
```

Comment:
Let's edit the `tailwind.config.js` so that these hard-coded numbers are in the configuration itself.